scripts/zones/Uleguerand_Range/mobs/Jormungand.lua

```diff
@@ -17,13 +17,24 @@ entity.onMobSpawn = function(mob)
     mob:setMobSkillAttack(0) -- resetting so it doesn't respawn in flight mode.
     mob:setAnimationSub(0) -- subanim 0 is only used when it spawns until first flight.
 
-    mob:setMod(xi.mod.UFASTCAST, 99)
+    mob:addImmunity(xi.immunity.BIND)
+    mob:addImmunity(xi.immunity.LIGHT_SLEEP)
+    mob:addImmunity(xi.immunity.PARALYZE)
+    mob:addImmunity(xi.immunity.SILENCE)
+    mob:addImmunity(xi.immunity.TERROR)
```

Comment:
If these immunities are static, we can put them all in mob_pools.sql

Reply:
For HNM/NMs I want to start utilizing the lua binding so when someone looks at a lua file for that NM they can tell what exactly is on the NM instead of looking in 20 different sql files